packages/nimble-components/src/rich-text/mention-list-box/styles.ts

```diff
@@ -0,0 +1,18 @@
+import { css } from '@microsoft/fast-element';
+import { styles as dropdownStyles } from '../../patterns/dropdown/styles';
+import { focusVisible } from '../../utilities/style/focus';
+import { controlHeight, menuMinWidth } from '../../theme-provider/design-tokens';
+
+export const styles = css`
+    ${dropdownStyles}
+
+    :host(:hover)::after,
+    :host(${focusVisible})::after {
+        width: auto;
+    }
+    
+    .listbox {
+        min-width: ${menuMinWidth};
+        height: calc(5.5 * ${controlHeight})
```

Comment:
Guessing this should be max-height based on the other comment when few or no items exist

Reply:
Updated as per the suggestion.